src/LSST/M1M3/TS/TSPublisher.h

```diff
@@ -94,7 +81,7 @@ class TSPublisher final : public cRIO::Singleton<TSPublisher> {
 
     MTM1M3TS_logevent_logLevelC _logLevel;
 
-    Telemetry::FlowMeterThread *_flowMeterThread;
+    Telemetry::FlowMeterThread *_flow_meter_thread;
```

Comment:
You are defining _flow_meter_thread as a raw pointer and then using it as an array in the non-simulation code. This can only end in tears. You'll need to do something like this:
```suggestion
    std::array<Telemetry::FlowMeterThread*, 2> _flow_meter_thread;
```

And then loop over the array or something for constructing, starting, stopping, and deconstructing.